base/libgit2/libgit2.jl

```diff
@@ -484,7 +484,17 @@ function cat(repo::GitRepo, spec)
     end
 end
 
-""" git rev-list --count <commit1> <commit2> """
+"""
+    LibGit2.revcount(repo::GitRepo, fst::AbstractString, snd::AbstractString)
```

Comment:
are fst and snd significant names? maybe something more anonymous would make sense for the docstring if there's no reference to anything about the arg names